pkg/oci/utils.go

```diff
@@ -39,6 +39,16 @@ var (
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
 )
 
+type CompatOCIProcess struct {
```

Comment:
@jodh-intel I think @sboeuf is trying to implement a structure that would allow us to be both rc4 and rc5 compatible.
Those compat structures replace the `spec.Process` ones in the below calls, they are used.